### ISSUE TITLE
Workers could stick to more than one network

### ISSUE THREAD

**Issue by abyrd** [conveyal/r5#212]
Currently a worker can only work on one network at a time. It would be possible to have more than one network in cache at once, but there's no way for the worker to tell the broker it is ready to work on more than one network.

**Comment by abyrd** [comment]
The workers now supply an array of network and scenario IDs to the broker, so they can easily be associated with more than one network.

**Comment by ansoncfit** [comment]
With minutely billing of EC2 instances, it may not be worth the complexity of switching networks.

**Comment by abyrd** [comment]
Agreed, I’ve thought about this several times recently. The whole design of
the task queue system was predicated on hourly billing. We could simplify
and clarify it significantly now that we have minutely billing by having
workers attached to a single network.

Although we can clearly say we don’t want the complexity of loading
multiple networks at once, it may be worth keeping the ability to switch
networks when a new task is received. All it saves us though is the
instance spin up time. And distance table building, but we’re planning to
serialize and reuse those soon.
